internal/controller/controllers/preprovisioningimage_controller_test.go

```diff
@@ -233,6 +233,31 @@ var _ = Describe("PreprovisioningImage reconcile", func() {
 			Expect(c.Get(ctx, key, infraEnv)).To(BeNil())
 			Expect(infraEnv.ObjectMeta.Annotations[EnableIronicAgentAnnotation]).To(Equal("true"))
 		})
+		It("Adds the ironic IPv6 address to the ignition when the infraEnv has no cluster and is annotated for IPv6", func() {
+			metav1.SetMetaDataAnnotation(&infraEnv.ObjectMeta, infraEnvIPFamilyAnnotation, ipv6Family)
+			Expect(c.Create(ctx, infraEnv)).To(BeNil())
+			backendInfraEnv.ClusterID = ""
+			mockInstallerInternal.EXPECT().GetInfraEnvByKubeKey(gomock.Any()).Return(backendInfraEnv, nil)
+			mockInstallerInternal.EXPECT().UpdateInfraEnvInternal(gomock.Any(), gomock.Any(), gomock.Any()).
+				Do(func(ctx context.Context, params installer.UpdateInfraEnvParams, internalIgnitionConfig *string) {
+					Expect(params.InfraEnvID).To(Equal(*backendInfraEnv.ID))
+					Expect(*internalIgnitionConfig).Should(ContainSubstring(url.QueryEscape(ironicServiceIPs[1])))
```

Comment:
I made the assumption here that `ironicServiceIPs[0]` is an IPv4 address and that `ironicServiceIPs[1]` is an IPv6 address. Though it's not directly obvious at first glance in the test.

Could we have a comment here that points out that we are using IPv6 addresses here?

Reply:
The test name is "Adds the ironic IPv6 address to the ignition when the infraEnv has no cluster and is annotated for IPv6" do you really think more commentary is required?

Also the `ironicServiceIPs` variable is just a hardcoded list [here](https://github.com/openshift/assisted-service/pull/6686/files#diff-139e1f8c8d05d26082afdbf307f7e25ad325febb81fc20e6de3136b048daa885R100)

Reply:
I saw the title of the test and I knew of the hard coded IP slice.

I thought it would be more convenient for future readers of this code to not have to look up the hard coded list.